point_quantity: 6

# x: [0.647, 3.594, 3.334, 2.517, 0.937, 0.196]
# y: [3.621, 2.809, 0.995, -1.903, -2.261, 0.229]
# yaw: [3.082, -1.437, 1.502, -2.889, 1.574, -0.066]
# poses_status: [true, false, true, false, false, true]

x: [3.542, 3.595, 1.573, -0.632, -1.284, -0.088]
y: [0.215, -2.644, -2.870, -3.079, -0.012, 0.184]
yaw: [1.674, -3.022, 0.133, 1.853, 1.705, -1.421]
poses_status: [true, false, true, false, false, true]
